cli/flox-rust-sdk/src/models/lockfile.rs

```diff
@@ -209,7 +209,7 @@ pub struct LockedPackageCatalog {
     // region: added fields
     pub system: System, // FIXME: this is an enum in the generated code, can't derive Arbitrary there
     pub group: String,
-    pub priority: usize,
+    pub priority: u64,
```

Comment:
nonblocking suggestion: you have an excellent commit message explaining this change - is it worth adding that as a comment? Maybe I just need to get better at checking `git  log` but I feel like personally I'm way more likely to read a comment

Reply:
I'll add a comment